index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/main.tsx"></script>
```

Comment:
Any reason this isn’t in `<head>` with `defer`? I’m not familiar with TypeScript, so don’t know if my approach actually works.

Reply:
This is how it was generated by the template. I did try moving it into the `<head>` and it still worked fine.